tracer/src/Datadog.Trace/Ci/CIEnvironmentValues.cs

```diff
@@ -777,6 +779,11 @@ private void SetupBitriseEnvironment()
             PipelineUrl = EnvironmentHelpers.GetEnvironmentVariable("BITRISE_BUILD_URL");
 
             Message = EnvironmentHelpers.GetEnvironmentVariable("BITRISE_GIT_MESSAGE");
+            AuthorName = EnvironmentHelpers.GetEnvironmentVariable("GIT_CLONE_COMMIT_AUTHOR_NAME");
+            AuthorEmail = EnvironmentHelpers.GetEnvironmentVariable("GIT_CLONE_COMMIT_AUTHOR_EMAIL");
+            CommitterName = EnvironmentHelpers.GetEnvironmentVariable("GIT_CLONE_COMMIT_COMMITER_NAME");
+            commiterEmail = EnvironmentHelpers.GetEnvironmentVariable("GIT_CLONE_COMMIT_COMMITER_EMAIL");
+            CommitterEmail = !string.IsNullOrWhiteSpace(commiterEmail) ? commiterEmail : CommitterName;
```

Comment:
Maybe I'm just really in love with switch expressions, but this could be a good place to use one, since I initially was confused that there were two lines beginning with "committerEmail"